addon/components/frost-date-picker.js

```diff
@@ -1,5 +1,7 @@
 import Ember from 'ember'
 
-export default Ember.Component.extend({
+const {Component} = Ember
```

Comment:
my understanding is not yet but because supporting is coming, this 'const' line is to allow easier switching when support does arrive.